test/README.md

```diff
@@ -3,7 +3,13 @@ and are here relicensed by their original author (Eemeli Aro) under the Unicode
 
 These test files are intended to be useful for testing multiple different message processors in different ways:
 
-- `syntax-errors.json` — An array of strings that should produce a Syntax Error or a Data Model Error when parsed.
+- `syntax-errors.json` — An array of strings that should produce a Syntax Error when parsed.
+
+- `data-model-errors.json` - An array of pairs of error names and arrays of strings:
+    - `error: string` (required) - The name of the error; names are defined in
+        ["MessageFormat 2.0 Errors"](https://github.com/unicode-org/message-format-wg/blob/main/spec/errors.md)
+        in the spec.
+    - `tests: string[]` (required): An array of strings that should produce `error` when processed.
```

Comment:
Why not an object `{ [error: string]: string[] }` as in `test-function.json`?

Reply:
No good reason -- changed it in [e3139f1](https://github.com/unicode-org/message-format-wg/pull/708/commits/e3139f1a162b8c3500b005fb2fd2eb9de2b0029d) to the format you suggested.